<?xml version="1.0"?>
<sdf version="1.7">
  <model name="plate_8in">
    <!--
    The frame of the plate has its +z axis pointing up (sticking out of the
    food). The origin-point of the plate-frame is along the plate's symmetry
    axis. If the plate was placed on a flat table, the origin (x=0, y=0, z=0)
    would be touching the table.  All vertices in the plate have a z-value
    greater than or equal to zero.  (Note that due to rounding errors a few
    vertices are approximately a micron below zero.)
    -->
    <link name="plate_8in">
      <pose>0 0 0 0 0 0</pose>
      <inertial>
        <pose>0 0 0.00915 0 0 0</pose>
        <mass>0.375</mass>
        <inertia>
          <ixx>0.000860</ixx>
          <ixy>0</ixy>
          <ixz>0</ixz>
          <iyy>0.000860</iyy>
          <iyz>0</iyz>
          <izz>0.00171</izz>
        </inertia>
      </inertial>
      <visual name="visual">
        <pose>0 0 0 0 0 0</pose>
        <geometry>
          <mesh>
            <uri>plate_8in_col.obj</uri>
            <scale>1.0 1.0 1.0</scale>
          </mesh>
        </geometry>
      </visual>
      <collision name="collision">
        <pose>0 0 0 0 0 0</pose>
        <geometry>
          <mesh>
            <uri>plate_8in_col.obj</uri>
            <scale>1.0 1.0 1.0</scale>
          </mesh>
        </geometry>
        <drake:proximity_properties>
          <drake:rigid_hydroelastic/>
          <!--
          We do not set drake:mesh_resolution_hint here because it has no
          effect on meshes imported from obj file.
          -->
        </drake:proximity_properties>
      </collision>
    </link>
    <frame name="origin">
      <pose relative_to="plate_8in">0 0 0 0 0 0</pose>
    </frame>
  </model>
</sdf>
